src/Utilities/Compiler/NullableAttributes.cs

```diff
@@ -1,6 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We likely want to switch to the .NET Foundation licenses in a follow-up PR.

Reply:
Partly relates to #3334